### PR TITLE
Don't whipe the disappearing messages timer on group updates

### LINKED ISSUE #172: group updates whipe the disappearing messages timer
`HandleGroupUpdateMessage` always sets the timer to 0.

woops.

### PR DESCRIPTION
**Pull request by Trolldemorted**
fixes #172

### CODE REVIEW

File: Signal-Windows.Lib/Storage/DB.cs [hunk 1]
```diff
@@ -1274,7 +1274,7 @@ public static SignalGroup InsertOrUpdateGroupLocked(string groupId, string displ
                             AvatarFile = avatarfile,
                             UnreadCount = 0,
                             CanReceive = canReceive,
-                            ExpiresInSeconds = expiresInSeconds,
+                            ExpiresInSeconds = 0,
```
Comment (golf1052) [inline Signal-Windows.Lib/Storage/DB.cs]: Should this be 0?

Comment (Trolldemorted) [reply Signal-Windows.Lib/Storage/DB.cs]: This happens in we hear about a group the first time - we have no clue what the expiry timer could be, so 0 is the most reasonable guess. We guessed 0 before too, just outside of this function in IncomingMessages.cs

Comment (golf1052) [reply Signal-Windows.Lib/Storage/DB.cs]: Ok, the `Update` part of the method name was why I was worried. What happens if the expiration time changes for a group, we won't be able to update it anymore?

Comment (Trolldemorted) [reply Signal-Windows.Lib/Storage/DB.cs]: For reasons unknown to me signal's group updates do not contain the timer (see #94). The expiration timer update handler uses SignalDBContext.UpdateExpiresInLocked to update the timer only

**Pull request merged**